### DISCUSSION TITLE
Workflow Persistence

### DISCUSSION THREAD

**Discussion by ashishkanojiay999** [elsa-workflows/elsa-core#4670] [Q&A]
Hi Team,
I want to know about how to store Workflow using Entity Framework Core provider and MSSQLServer DB. How to store workflow in MSSQL Server DB table?

**Comment by Snotax** [comment]
You can inject it like this:

```
       .UseEntityFrameworkPersistence(ef =>
       {
           ef.UseSqlServer(configuration.GetConnectionString("Workflow-DB")!);
       }, true))
```
       
&& for Quartz
       
```
          .AddQuartzTemporalActivities(configureQuartz: quartz => quartz.UsePersistentStore(store =>
        {
            store.UseJsonSerializer();
            store.UseSqlServer(providerOptions =>
            {
                providerOptions.ConnectionString = configuration.GetConnectionString("Workflow-DB") ?? string.Empty;
                providerOptions.ConnectionStringName = "Workflow-DB";
            }, "Workflow-DB");
            store.UseClustering();
        }))
```

**Comment by ashishkanojiay999** [comment]
Hi, I tried the same thing but getting below issue

A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)'

**Comment by Snotax** [comment]
That doesn't sound like an ELSA issue. Are you sure the servers can connect to each other?

**Comment by sfmskywalker** [comment]
@Snotax is correct, it sounds like there's an issue with your connection string and/or server.
